change/@fluentui-react-switch-dbade308-9edd-4348-9e8d-a6909fcb2a9c.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Implementing useCapture and useEvent.",
```

Same comment as above
```suggestion
  "comment": "Adopting useCapture and useEvent utilities.",
```